content/07-native-tokens/03-getting-started.mdx

```diff
@@ -55,7 +55,7 @@ cardano-cli ...
 
 ### Understanding values
 
-cardano-node version 1.35.0 and later of the cli no longer supports ASCII token names, only hexidecimal format is support. The result of running  
+cardano-node version 1.35.0 and later of the cli no longer supports ASCII token names, only hex-encoded format is supported. The result of running  
```

Comment:
This isn't new, I think asset names have been ASCII-only since at least 1.32, maybe earlier.

Reply:
This is not resolved, I suggest changing the version to 1.32.1 because this isn't a new change in 1.35.0